make.sh

```diff
@@ -8,13 +8,14 @@ NGINX_BUILD_DIR="$BUILD_DIR/nginx-$NGINX_VERSION"
 
 function clean {
     echo "Cleaning ..."
-    rm -rf "$BUILD_DIR"
-    mkdir -p "$BUILD_DIR"
+    rm -rf "$BUILD_DIR" || true
 }
 
 function setup {
     echo "Setting up ..."
+    mkdir -p "$BUILD_DIR"
     tar xzf "nginx-$NGINX_VERSION.tar.gz" -C "$BUILD_DIR"
+    sbt update
```

Comment:
what's this for?

Reply:
The goal is to ask sbt to download dependencies. When building inside docker the ivy cache is empty by default which causes dependencies to be downloaded every time the build is invoked. By explicitly decoupling this, we allow docker to cache the dependencies for faster builds.

Reply:
Cool, thanks